crypto/rand/rand_lcl.h

```diff
@@ -17,21 +17,24 @@
 # include <openssl/ec.h>
 # include "internal/rand.h"
 
-/* we require 256 bits of randomness */
-# define RANDOMNESS_NEEDED (256 / 8)
+/* Amount of randomness (in bytes) we want for initial seeding. */
+# define RANDOMNESS_NEEDED              (128 / 8)
```

Comment:
Paul says this is fine, he's a cryptographer and I'm not.  You disagree.  Ok, if you want to open an other PR to change the value, feel free.

It might be more productive to have a discussion about this on openssl-dev.